docs/en-US/component/image.md

```diff
@@ -69,7 +69,7 @@ image/image-preview
 | scroll-container      | the container to add scroll listener when using lazy load. By default, the container to add scroll listener when using lazy load.                 | ^[string] / ^[object]`HTMLElement`                                      | —       |
 | alt                   | native attribute `alt`.                                                                                                                           | ^[string]                                                               | —       |
 | referrerpolicy        | native attribute [referrerPolicy](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/referrerPolicy).                              | ^[string]                                                               | —       |
-| crossorigin        | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                              | ^[enum]`'' \| 'anonymous' \| 'use-credentials'` | 'anonymous'                                                             | —       |
+| crossorigin        | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                              | ^[enum]`'' \| 'anonymous' \| 'use-credentials' \| 'null'`               | 'anonymous'                                                             | —       |
```

Comment:
Don't change this, please clear the default value of `crossorigin`.

Reply:
https://github.com/element-plus/element-plus/blob/dev/packages/components/image/src/image.ts#L111C26-L111C26

Please remove this value.